pages/Configuring/Dwindle-Layout.md

```diff
@@ -32,22 +32,22 @@ container get removed, the group will be broken back to its dwindle form.
 
 category name: `dwindle`
 
-| name | description | type | default |
-|---|---|---|---|---|
-| pseudotile | enable pseudotiling. Pseudotiled windows retain their floating size when tiled. | bool | false |
-| col.group_border | inactive (out of focus) group border color | color | 0x66777700 |
-| col.group_border_active | active group border color | color | 0x66ffff00 |
-| force_split | 0 -> split follows mouse, 1 -> always split to the left (new = left or top) 2 -> always split to the right (new = right or bottom) | int | 0 |
-| preserve_split | if enabled, the split (side/top) will not change regardless of what happens to the container. | bool | false |
-| special_scale_factor | 0 - 1 -> specifies the scale factor of windows on the special workspace | float | 0.8 |
-| split_width_multiplier | specifies the auto-split width multiplier | float | 1.0 |
-| no_gaps_when_only | whether to apply gaps when there is only one window on a workspace, aka. smart gaps. | bool | false |
-| use_active_for_splits | whether to prefer the active window or the mouse position for splits | bool | true |
-
-## Bind Dispatchers
-
-| dispatcher | description | params |
-|---|---|---|
-| togglegroup | toggles the current window and its siblings (recursively) into a group | none |
-| changegroupactive | switches to the next window in a group. | b - back, f - forward. |
-| togglesplit | toggles the split (top/side) of the current window | none |
+| name                    | description                                                                                                                        | type  | default    |
```

Comment:
I asked you to revert this.

Reply:
And we have. Once again, someone (*cough* @0x4nt *cough* fucked up.)